### PR TITLE
SFD: Certificates are also on the Profile page

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3762](https://openedx.atlassian.net/browse/DOC-3762)

In the certificates doc for learners, mention that certificates can also be found on the Profile page. I didn't modify the PDF certs section, since we hope nobody is looking for those any longer.

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Doc team review (sanity check, copy edit, or dev edit?): @catong 

### Testing

- [x] Ran ./run_tests.sh without warnings or errors

### Post-review

- [x] Squash commits

### CODE REVIEW

File: en_us/students/source/SFD_certificates.rst [hunk 1]
```diff
@@ -177,7 +180,8 @@ immediately after you receive the verified certificate for the final course in
 the program.
 
 The XSeries and MicroMasters program certificates that you have earned are
-visible on one side of the **Programs** tab on your dashboard.
+visible on one side of the **Programs** tab on your dashboard and in your
```
Comment (catong) [inline en_us/students/source/SFD_certificates.rst]: It was in existing doc, but I'd strike "on one side of" - not necessary.

File: en_us/students/source/SFD_certificates.rst [hunk 2]
```diff
@@ -177,7 +180,8 @@ immediately after you receive the verified certificate for the final course in
...
+**Profile** page.
```
Comment (catong) [inline en_us/students/source/SFD_certificates.rst]: "in" (your profile page) -> "on"

**Comment by catong** [comment]
@grantgoodmanedX Minor nits on the changes here. 

There might also be updates needed to the sections on sharing/printing certs - probably just to where certs can be accessed (not just the dashboard). I have not experimented with clicking certs on my profile to check the behavior. http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/SFD_certificates.html#share-your-certificates-on-social-media

File: en_us/students/source/SFD_certificates.rst (hunk 2 above)
Comment (grantgoodmanedX) [reply en_us/students/source/SFD_certificates.rst]: I made these edits.

**Pull request merged**